cli/src/cli/test.rs

```diff
@@ -109,7 +143,10 @@ pub mod tests {
             .exists());
     }
 
+    // TODO: this test is non-deterministic, pivoting around the fact that the
+    // behaviour of the command is different, depending whether nodejs is installed on the system or not
     #[test]
+    #[cfg(not(windows))]
```

Comment:
@Connoropolous for future reference please use `#[cfg(feature = "broken-tests")]` as we might never fix `#[cfg(not(windows))]` flags